modules/ROOT/pages/streaming-about.adoc

```diff
@@ -271,6 +248,17 @@ increments at 100 objects per increment and allows a maximum buffer size of
  </sfdc:query>
 ----
 
+[[disable_repeatable_streaming]]
+== Disabling Repeatable Streaming
+
+You can disable repeatable streaming through the `non-repeatable-stream` and `non-repeatable-iterable` strategies. The strategy to use depends on the type of stream.
+
+Use this option only if you are certain that there is no need to consume the stream several times and only if you need a very tight optimization for performance and resource consumption. 
```

Comment:
If you want to emphasize this even more, try list format:
```suggestion
Use this option only if both of the following are true:
* You are certain that there is no need to consume the stream several times.
* You need tight optimization for performance and resource consumption. 
```
You might also consider whether it is more effective to say there is not need to consume several times, or the stream will be consumed only one or a few times.